benchmarks/runners/qcdnum_bench.py

```diff
@@ -27,7 +27,7 @@ def benchmark_lo(self):
     def benchmark_nlo(self):
         self.run(
             [{"PTO": 1}],
-            observables.build(**(observables.default_config[1])),
+            observables.build(**(observables.default_config[2])),
```

Comment:
We're using `observables.default_config` to store the default configuration for any perturbative order, so the number it's not just a random label, but it's meant to be PTO.

The idea of saving other configurations it's not bad, simply make a new dictionary in `observables` module, and try to make a sensible choice (a general one you can reuse more than once, and then update the few things that are different).